pgcli/pgcompleter.py

```diff
@@ -185,6 +190,15 @@ def get_completions(self, document, complete_event, smart_completion=None):
                 tables = suggestion['tables']
                 _logger.debug("Completion column scope: %r", tables)
                 scoped_cols = self.populate_scoped_cols(tables)
+
+                if suggestion.get('drop_unique'):
+                    # drop_unique is used for 'tb11 JOIN tbl2 USING (...'
+                    # which should suggest only columns that appear in more than
+                    # one table
+                    scoped_cols = [col for (col, count)
+                                         in Counter(scoped_cols).items()
+                                           if count > 1 and col != '*']
```

Comment:
I was thinking that using a `set` and doing an intersection would produce the same result, but I presume the code won't be as simple. Is that right?